fastcrypto/Cargo.toml

```diff
@@ -17,7 +17,7 @@ eyre = "0.6.8"
 hex = "0.4.3"
 hkdf = { version = "0.12.3", features = ["std"] }
 rand = { version = "0.8.5", features = ["std"] }
-rust_secp256k1 = { version = "0.24.0", package = "secp256k1", features = ["recovery", "rand-std", "bitcoin_hashes", "global-context"] }
+rust_secp256k1 = { version = "0.24.1", package = "secp256k1", features = ["recovery", "rand-std", "bitcoin_hashes", "global-context"] }
```

Comment:
Wow it solves a very surprising issue re serialization.

https://github.com/rust-bitcoin/rust-secp256k1/issues/491